doc/include/api_stdrt_cproc.inc

```diff
@@ -1,159 +1,106 @@
 
 [list_begin definitions]
 [comment {% % %% %%% %%%%% %%%%%%%% %%%%%%%%%%%%%}]
-[def Tcl_Obj*]
-[def object]
-
-The function returns a value of type [type Tcl_Obj*].
-
-This value becomes the interpreter result, if not [const 0]. The Tcl
-status is [const TCL_ERROR] when a [const 0] is returned, and
-[const TCL_OK] otherwise.
-
-[para][emph Note] how it is the responsibility of the function to set
-the interpreter result to an appropriate error message when returning
-[const 0].
-
-[para][emph Attention]: The conversion assumes that the value belonged
-to the function, with an associated reference count, and decrements
-the reference count to indicate the loss of ownership by the
-function. This means that it is an error to return a value whose
-reference count is zero. The system will crash at some point after the
-return due to reuse of the freed memory.
+[def {Tcl_Obj* object}]
```

Comment:
The original form using two `def` commands was used to make it clear that we are looking at two separate type names.
In the new form this is not as clear. Maybe add a comma ? I.e. `[def {Tcl_Obj*, object}] ?

Reply:
With the following `Tcl_Obj*0`, `object0` sequence the old form was kept. Revert this to old form as well.

Reply:
Done with commit 29380521d0873ea1dcc5f8cdff6742b20764473f (Branch `documentation` in my repo, not seen by the PR).